liblxqt-config-cursor/CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ target_link_libraries(lxqt-config-cursor
 
 install(TARGETS
     lxqt-config-cursor
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}"
```

Comment:
@agaida Sorry for the delay. Real life. 
This causes an  error while loading shared libraries. The dynamic linker (default configuration) will not find the library there.
Filled an issue: https://github.com/lxde/lxqt/issues/793